### PR TITLE
MPOL modification request

### PR DESCRIPTION
**Pull request by gopex21**
Modified the json to add our second relay

### CODE REVIEW

**Review by cxhercules — CHANGES_REQUESTED** [review]
Please rebase or pull most resent copy and start from there.

File: spo-links.json [hunk 1]
```diff
@@ -151,6 +151,10 @@
       "FAIR": { "links": [ 
         { "url": "https://eu01.submit.fairpool.eu/api/submit/tx", "location": "Austria" }
       ] },
+      "MPOL": { "links": [ 
+        { "url": "http://217.28.30.43:8090/api/submit/tx", "location": "Canada" },
```
Comment (cxhercules) [inline spo-links.json:155]: Also tested both endpoints and this one returns this error:
```
Failed
400:
"transaction submit error ShelleyTxValidationError ShelleyBasedEraAlonzo (ApplyTxError [UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (ValueNotConservedUTxO (Value 0 (fromList [])) (Value 9831683 (fromList []))))),UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (BadInputsUTxO (fromList [TxInCompact (TxId {_unTxId = SafeHash \"a05e800cc2a9e35af273bb774c4ed9b6f70a12077334c4d744ff04987df0e909\"}) 0,TxInCompact (TxId {_unTxId = SafeHash \"a05e800cc2a9e35af273bb774c4ed9b6f70a12077334c4d744ff04987df0e909\"}) 1]))))])"
```

Comment (cxhercules) [reply spo-links.json:155]: Please check the health of this endpoint and confirm.

**Comment by gopex21** [comment]
Hi,

Node was restored and is currently synchronizing.

Will modify the most recent copy

**Pull request closed without merging**